Lib/gettext.py

```diff
@@ -164,26 +164,15 @@ def _parse(tokens, priority=-1):
     return result, nexttok
 
 
-def _as_int(n):
-    try:
-        i = round(n)
-    except TypeError:
-        raise TypeError('Plural value must be an integer, got %s' %
-                        (n.__class__.__name__,)) from None
-    import warnings
-    warnings.warn('Plural value must be an integer, got %s' %
-                  (n.__class__.__name__,),
-                  DeprecationWarning, 4)
-    return n
-
-
 def c2py(plural):
     """Gets a C expression as used in PO files for plural forms and returns a
     Python function that implements an equivalent expression.
     """
+    import textwrap
```

Comment:
I do not think there is a necessary to add this new dependency.

Reply:
Ok, removed.